### PR TITLE
Problem Suggestion: Add "Close Tuples (hard version)" to Silver - Binary Search

### PR DESCRIPTION
**Pull request by jeffkmeng**
User `7AF1MKJjqmXfBDGhimSWEhXqe8k1` suggested adding the problem [Close Tuples (hard version)](https://codeforces.com/problemset/problem/1462/E2) to the `general` table of the module [Silver - Binary Search](https://usaco.guide/silver/binary-search#problemlist-general).

**Automatically Generated JSON:**
```json
{
  "uniqueId": "cf-1462E2",
  "name": "Close Tuples (hard version)",
  "url": "https://codeforces.com/problemset/problem/1462/E2",
  "source": "CF",
  "difficulty": "Normal",
  "isStarred": false,
  "tags": [
    "Sorting",
    "Math",
    "Binary Search on Sorted Array"
  ],
  "solutionMetadata": {
    "kind": "autogen-label-from-site",
    "site": "CF"
  }
}
```
**Additional Notes**: None

*This PR was automatically generated from a user-submitted problem suggestion on the USACO guide.*

### CODE REVIEW

**Review by jessechoe10 — APPROVED** [review]

File: content/3_Silver/Binary_Search.problems.json [hunk 1]
```diff
@@ -242,6 +242,19 @@
...
       }
     },
+    {
+      "uniqueId": "cf-1462E2",
+      "name": "Close Tuples (hard version)",
+      "url": "https://codeforces.com/problemset/problem/1462/E2",
+      "source": "CF",
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": ["Sorting", "Math", "Binary Search on Sorted Array"],
```
Comment (maggie-j-liu) [inline content/3_Silver/Binary_Search.problems.json:252]:
```suggestion
      "tags": ["Sorting", "Math", "Binary Search"],
```

**Comment by SansPapyrus683** [comment]
I mean, to be honest I think this would be more suited for the Modular Arithmetic section. It requires knowledge of modular inverses, which is covered in Gold.

**Comment by jessechoe10** [comment]
> I mean, to be honest I think this would be more suited for the Modular Arithmetic section. It requires knowledge of modular inverses, which is covered in Gold.

No it doesn't :laughing:.

**Comment by SansPapyrus683** [comment]
yes-yes it does???

**Comment by jessechoe10** [comment]
> yes-yes it does??? how else would you do modular division?

[My code](https://forum.usaco.guide/t/closed-tuples/1273/2?u=jessechoe10)

**Comment by SansPapyrus683** [comment]
cool, but maybe we should still add "modular arithmetic" in the tags

**Comment by thecodingwizard** [comment]
Does that pass in Java? If not we probably shouldn't add it... (Or we should move to gold instead)

**Comment by jessechoe10** [comment]
> Does that pass in Java? If not we probably shouldn't add it... (Or we should move to gold instead)

Idk

**Comment by jessechoe10** [comment]
We should probably remove then...

**Pull request closed without merging**